openedx/core/lib/api/tests/test_permissions.py

```diff
@@ -176,3 +186,12 @@ def test_has_permission_as_non_owner(self):
         request = RequestFactory().get('/?username={}'.format(user.username))
         request.user = UserFactory.create()
         self.assertFalse(self.permission.has_permission(request, None))
+
+    def test_has_permission_with_view_kwargs_as_non_owner(self):
+        """ Non-owners should not have permission. """
+        user = UserFactory.create()
+        request = RequestFactory().get('/')
+        request.user = user
+        view = GenericAPIView()
```

Comment:
we should DRY these lines into a single helper method and call it in both test methods.